frontend/src/modules/projects/board.tsx

```diff
@@ -33,6 +42,11 @@ export default function Board() {
     projects.filter((p) => !p.archived).sort((a, b) => sortById(a.id, b.id, submenuItemsOrder[workspace.id])),
   );
 
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [containerWidth, setContainerWidth] = useState(() => containerRef.current?.clientWidth ?? 0);
+  const scrollerWidth = getScrollerWidth(containerWidth, projects.length);
+  const panelMinSize = typeof scrollerWidth === 'number' ? (PANEL_MIN_WIDTH / scrollerWidth) * 100 : 100 / (projects.length + 1); // + 1 so that the panel can be resized to be bigger or smaller
```

Comment:
Let me know if this logic is hard to understand, we might be able to make it more understandable.